.github/workflows/tests-unit.yml

```diff
@@ -5,22 +5,26 @@ on: [push]
 jobs:
   build:
 
-    name: Test on node ${{ matrix.node_version }} and ${{ matrix.os }}
```

Comment:
The previous version failed because we weren't using the matrix but trying to reference it in the name.